assets/css/2-components/13-package-group.css

```diff
@@ -53,7 +53,7 @@
 }
 
 .package-group:hover {
-	background-color: (--focused-background);
+	background-color: var(--focused-background);
```

Comment:
I'm quite shocked this wasn't caught by anything! :O

Reply:
Caught by my browser devTools ❤️